README.md

```diff
@@ -13,6 +13,25 @@ For reading files, it provides multiple levels of support:
 
 Supported file formats: ELF, Mach-O, Windows PE/COFF, Wasm, and Unix archive.
 
+## Example
+```rust
+use std::fs::File;
+use std::io::Read;
+use object::{Object, ObjectSection, File as ObjectFile};
+
+/// Reads an ELF-file and displays the content of the ".boot" section.
+fn main() {
+    let mut file = File::open("./multiboot2-binary.elf").unwrap();
+    let mut data = vec![];
+    file.read_to_end(&mut data).unwrap();
+    let data = data.into_boxed_slice();
```

Comment:
```suggestion
    let data = fs::read("<path-to-file>")?
```

Reply:
Oh great! After almost 3 years of doing Rust, I didn't know about `fs::read` and always did the other, more laborious way - thanks!